msu/msu_mod/setup_msu_mod.nut

```diff
@@ -0,0 +1 @@
+::MSU.Mod <- ::MSU.registerMod(::MSU.ID, ::MSU.Version, ::MSU.Name);
```

Comment:
We should discuss if we want this wrapper for returning the mod object or retain the original method of instantiating `::MSU.Class.Mod`

Reply:
I'm more in favor of the original system because it's simpler but I don't wanna die on that hill

Reply:
The wrapper has been removed